lib/Doctrine/Common/Cache/MemcacheCache.php

```diff
@@ -74,7 +74,11 @@ protected function doFetch($id)
      */
     protected function doContains($id)
     {
-        return (bool) $this->memcache->get($id);
+        $flags = null;
+        $this->memcache->get($id, $flags);
+        
+        //if memcache has changed the value of "flags", it means the value exists
```

Comment:
Who we know that the flag will be unchnaged ? I don't see anything in phpdoc about that, I have value that does not exists and the flag is changing to 0